Sluggo/Models/AppIdentity.swift

```diff
@@ -43,7 +107,22 @@ class AppIdentity: Codable {
     }
     
     func saveToDisk() -> Bool {
-        guard let appIdentityData = JsonLoader.encode(object: self) else {
+        
+        let savedRecord = AppIdentity()
+        
+        savedRecord._pageSize = self._pageSize
+        savedRecord._baseAddress = self._baseAddress
```

Comment:
Base address may eventually be put directly on the login screen. We should decide if this would count as personally identifying information, since it could theoretically leak information about what teams the user is on.

(Or, frankly, consider nixing remember me, but that's beyond the scope of this branch of course.)

Reply:
This is necessary for app state. Both of these should be saved no matter what and then changed where necessary